### PR TITLE
Add patched version of ppx_tools due to ast changes with locals

### PR DESCRIPTION
**Pull request by Ekdohibs**

### CODE REVIEW

**Comment by kit-ty-kate** [comment]
* why add 2 versions `6.5` and `6.6` ?
* the change you're making seems to be only valid for flamabda2 compilers, however the opam files tells us that it is compatible with any versions of the compiler, not just the flambda2 compilers

File: packages/ppx_tools/ppx_tools.6.6+flambda2/opam [hunk 1]
```diff
@@ -0,0 +1,25 @@
+opam-version: "2.0"
+synopsis: "Tools for authors of ppx rewriters and other syntactic tools"
+maintainer: "Kate <kit.ty.kate@disroot.org>"
+authors: "Alain Frisch <alain.frisch@lexifi.com>"
+license: "MIT"
+tags: "syntax"
+homepage: "https://github.com/ocaml-ppx/ppx_tools"
+bug-reports: "https://github.com/ocaml-ppx/ppx_tools/issues"
+depends: [
+  "ocaml" {>= "4.08.0" & < "5.1.0"}
```
Comment (kit-ty-kate) [inline packages/ppx_tools/ppx_tools.6.6+flambda2/opam:10]: The following change is needed:
```suggestion
  "base-flambda2"
```

**Comment by Ekdohibs** [comment]
I think I added `ppx_tools.6.5` because at some point opam tried to install it automatically, but I think it was not up-to-date when I did that, I can remove it.

**Comment by kit-ty-kate** [comment]
Thanks

**Pull request merged**